scripts/etch-language-tests/ml/dataloader.etch

```diff
@@ -0,0 +1,3 @@
+function main()
+    var data_loader = DataLoader();
+endfunction
```

Comment:
I don't think this test is pulling its weight, as it's only exercising the constructor, especially given that the other tests in this PR construct an instance of a DataLoader. If there's nothing interesting one can do with a dataloader in isolation, I would get rid of this one.